companion/src/modeledit/setup.cpp

```diff
@@ -238,9 +238,9 @@ ModulePanel::ModulePanel(QWidget * parent, ModelData & model, ModuleData & modul
   lock = false;
 
   // a change will not register if only one item was added
-  if (ui->protocol->count() == 1)
+  if (ui->protocol->count() == 1) {
```

Comment:
Can I suggest that we document it (a wiki page ?) when we agree ? There is just too much for my little brain to remember :)